main.c

```diff
@@ -269,6 +275,8 @@ static void usage(const char *argv0)
 	printf("-m, --mtu=MTU            specify MTU (default=%d, max=65521)\n", DEFAULT_MTU);
 	printf("--cidr=CIDR              specify network address CIDR (default=%s)\n", DEFAULT_CIDR);
 	printf("--disable-host-loopback  prohibit connecting to 127.0.0.1:* on the host namespace\n");
+	printf("--use-netns-path	 replace PID with a network namespace path\n");
```

Comment:
I prefer `--netns-type=path`, but not strong opinion
https://github.com/rootless-containers/slirp4netns/issues/84#issuecomment-482490915